js/app.js

```diff
@@ -79,9 +77,36 @@ function render (Store) {
 
 };
 
-
 //salmon image
 var img = document.createElement('img');
 img.setAttribute('src', 'images/salmon.png');
 img.setAttribute('alt', 'cute picture of a salmon');
 article.appendChild(img);
+
+//for verifying whether store name is already in use in the table
+var storeNames = ['alki', 'first and pike', 'seatac airport', 'capital hill', 'seattle center'];
+
+var cookieCruncher = document.getElementById('addCookieCruncher');
+// event listeners ened to know: what event do they care about, and what do they want to do when it happens.
+cookieCruncher.addEventListener('submit',
```

Comment:
No matter what numbers I put in your form, I get back only 0 and 1 for the number of cookies sold.